src/Ractive/prototype/set.js

```diff
@@ -10,9 +10,9 @@ export default function Ractive$set ( keypath, value ) {
 	if ( isObject( keypath ) ) {
 		const map = keypath;
 
-		for ( keypath in map ) {
-			if ( map.hasOwnProperty( keypath) ) {
-				set( this, keypath, map[ keypath ] );
+		for ( const k in map ) {
+			if ( map.hasOwnProperty( k) ) {
+				set( this, k, map[k] );
 			}
```

Comment:
@heavyk IIRC I've tried that a while ago and the difference was rather small, but feel free to give it a shot.

@Rich-Harris we could actually have both speed and readability in this case, if we used our own `forEach` implementation.